frontend/src/pages/ProfilePage/ProfilePage.module.scss

```diff
@@ -0,0 +1,5 @@
+.userProfile {
```

Comment:
this is bad css - should use flexboxes to centre (like below) 
`.userProfile {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100%;
}
`